aidial_adapter_openai/app.py

```diff
@@ -129,56 +168,62 @@ async def call_chat_completion(
             creds,
             api_version,
             deployment_id,
+            app_config,
         )
-
-    if deployment_id in DALLE3_DEPLOYMENTS:
+    if deployment_id in app_config.DALLE3_DEPLOYMENTS:
         storage = create_file_storage("images", request.headers)
         return await dalle3_chat_completion(
             data,
             upstream_endpoint,
             creds,
             is_stream,
             storage,
-            DALLE3_AZURE_API_VERSION,
+            app_config.DALLE3_AZURE_API_VERSION,
```

Comment:
Try to place `app_config` as a first argument instead of the last.

The order of argument should reflect their scope _(which conveys certain information to the reader)_, rather than the chronology of their appearance in the code _(which is random and therefore of no interest)_.

Reply:
`app_config.DALLE3_AZURE_API_VERSION` represents only `api_version` here

We have quite common function signature across different models:
```
async def chat_completion(
    data: Any,
    upstream_endpoint: str,
    ...
```python

And not all of these handlers have `api_version` parameter

If I'll introduce `api_version` as first parameter for dalle, we'll have different signatures of `chat_completion`

I'd rather prefer not to do this

Reply:
I misplaced the comment - I meant all the places where a new `app_config` parameter was introduced as the **last** argument.